ftl/core/importing.ftl

```diff
@@ -67,6 +67,12 @@ importing-packaged-anki-deckcollection-apkg-colpkg-zip = Packaged Anki Deck/Coll
 importing-pauker-18-lesson-paugz = Pauker 1.8 Lesson (*.pau.gz)
 # the '|' character
 importing-pipe = Pipe
+# Warning displayed when the csv import preview table is clipped (some columns were hidden)
+importing-preview-truncated =
+    { $count ->
+        [one] Only the first column is shown. If this doesn't seem right, try changing the field separator.
```

Comment:
When first reading this, I assumed you were repurposing this message to also warn the user when only a single column in shown, as it's not bad advice then either.

If this was just for the translation system, you could probably even drop the [one] case, and mention in the translation comment that $count will only ever be 1000 or a similarly-large number, which would allow translators to skip zero/one/few cases.

On the other hand, if you were thinking of using this in the 1 case as well in the future, I'd recommend rewording it so it can make use of $count, as hard-coding a number instead of using a variable in the one case has tripped up translators in the past.